### PR TITLE
ChartView

### PR DESCRIPTION
**Pull request by bridges-wood**
Add harness for chart viewing.

### CODE REVIEW

File: src/main/java/tempest/ui/pages/HomePage.java [hunk 1]
```diff
@@ -13,19 +12,21 @@
...
     public HomePage(GUIManager guiManager) {
         super();
         this.manager = guiManager;
+        addNavButtons();
+    }
 
+    private void addNavButtons() {
         JPanel buttonPanel = new JPanel();
-        manageModulesButton = new LinkButton("Add a new module", PageNames.MANAGE_MODULES, this);
-        addSessionButton = new LinkButton("Add a new session", PageNames.ADD_SESSION, this);
```
Comment (Drayz64) [inline src/main/java/tempest/ui/pages/HomePage.java:25]: Any particular reason for having the LinkButtons initialised at the top? I'm assuming just your preference

Comment (Drayz64) [reply src/main/java/tempest/ui/pages/HomePage.java:25]: I guess it is a bit neater in the method

**Review by Drayz64 — APPROVED** [review]
As long as you don't mind the changes I made, all is good :)

**Comment by bridges-wood** [comment]
> As long as you don't mind the changes I made, all is good :)

Nice changes 💯

**Pull request merged**